designs/strings-design.md

```diff
@@ -0,0 +1,227 @@
+# Week 8 Update
+We have flushed out our design and managed to pass all the tests. The output of the tests are shown below. The updated design section is shown after the tests.
+## TEST CASES and output:
+* test case 1: var definition for string (use asserttc to check)
+```
+s:str = "asdf"
```

Comment:
Make sure you start with primitive tests that are running functions on string literals, such as `print("abcd")` or `len("abcd")`

Reply:
I agree that our test cases could be organized better. However, since we were able to pass all the test cases I believe the ordering matter less now.